### DISCUSSION TITLE
git-clone creates a detached head

### DISCUSSION THREAD

**Discussion by Manfred73** [tektoncd/catalog#1021] [General]
I'm using the git-clone ClusterTask in my pipeline, but found that this creates a detached head.
I used a mvn comman to update the pom version of cloned branch and then tried to commit this, using git add, git commit and git push, but this doesn't work because of detached head. How should we handle this when using git-clone.

In the same way I would like to create a tag in the git repository with the same version. How would I create a git tag after I have done a git-clone. 
On Tekton Hub I found two release tasks which can create tags, but these are only for gitlab and github. We're using bitbucket.

**Comment by darktempla** [comment]
Any update on this one?
Is there a way to get the `git-clone` task to checkout the code with detaching it.
Looking at the parameters it exposes it does not seem obvious.
